addon-test-support/table-manager.ts

```diff
@@ -20,8 +20,11 @@ export const buildColumnDefinition = (key: string, extra?: { [key: string]: any
     category: '',
     size: FieldSize.Medium,
     orderable: false,
+    orderable_by: [],
     filterable: false,
-    facetable: false
+    filterable_by: [],
+    facetable: false,
```

Comment:
Could we remove any dependency to `facetable`, `filterrable` or `orderable`?  We're going to ditch this asap in the back.

Reply:
Let's make this my TP con for tomorrow okay ? 😂

Reply:
as you wish. Just give me the green light.